Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN git clone https://github.com/kubeflow/kubeflow.git && \
     git checkout d1da825
 
 # --- Build the backend kubeflow-wheel ---
-FROM python:3.7-slim-buster AS backend-kubeflow-wheel
+FROM python:3.7-slim AS backend-kubeflow-wheel
```

Comment:
Cross posting the same question with https://github.com/kubeflow/kubeflow/pull/6650#discussion_r1029094207

Why do we remove `buster`?

Reply:
See https://github.com/kubeflow/kubeflow/pull/6650#discussion_r1029567652:

> It's "more multi-arch" than the alternative (see screenshot below) & based on our tests, it works equally good.